resume-buddy/src/main/webapp/style.css

```diff
@@ -0,0 +1,52 @@
+body,
+h2,
+h3,
+h4,
+h5,
+h6,
+.w3-wide {
+  font-family: "Roboto", sans-serif;
+}
+ 
+.title {
+  font-family: 'Noto Sans TC', sans-serif;
+  font-size: 100px;
+  align-items: center;
+  text-align: center;
+  }
+ 
+.logo {
+  width: 130px;
+  height: 130px;
+  float: right;
+  margin-right: 50px;
+  margin-top: 10px;
+  margin-bottom: 10px;
```

Comment:
Nit: you can also use shorthand properties https://google.github.io/styleguide/htmlcssguide.html#Shorthand_Properties